app/controllers/root_controller.rb

```diff
@@ -32,6 +32,8 @@ class RootController < ApplicationController
     gem_layout_full_width_explore_header
     gem_layout_no_feedback_form
     gem_layout_no_footer_navigation
+    gem_layout_old_header
```

Comment:
I think my changes [here](https://github.com/alphagov/static/pull/2595/commits/0525c73d4b95a1948c9428c8edb800fb0000a3a3)  mean this would become duplicates?

Reply:
Duplicate removed and the templates alphabetised 👍